pkg/cri/sbserver/container_create.go

```diff
@@ -730,9 +730,8 @@ func (c *criService) buildWindowsSpec(
 	extraMounts []*runtime.Mount,
 	ociRuntime config.Runtime,
 ) (_ []oci.SpecOpts, retErr error) {
-	specOpts := []oci.SpecOpts{
-		customopts.WithProcessArgs(config, imageConfig),
-	}
+	var specOpts []oci.SpecOpts
+	specOpts = append(specOpts, customopts.WithProcessCommandLineOrArgs(config, imageConfig))
```

Comment:
If the function is only intended for use on Windows, then it would be better to add Windows to the name of the function.

Reply:
I could change this but CommandLine itself is a windows only spec right - https://github.com/opencontainers/runtime-spec/blob/main/specs-go/config.go#L46 ? Doe WithProcessCommandLineOrArgsForWindows() sound better?

Reply:
@dmcgowan ping on this :)

Reply:
Renaming sounds better if it is truly something only for Windows. Removing the panic would be good as well, we don't normally have panics in the runtime path to represent not implemented,  unless its on startup.